### DISCUSSION TITLE
Eigen strain function of nodal coordinate

### DISCUSSION THREAD

**Discussion by devesh-tiwari** [idaholab/moose#23990] [Q&A Modules: Phase field]
Hello, I want to use a nodal coordinate dependent eigen strain in my phase field model (e^(-kx) where s is the position along x axis. Kindly guide me about how to access the nodal coordinate

**Comment by GiudGiud** [comment]
Hello

I m not sure what you mean here. 
You need to get a coordinate from a mesh that meets a certain criterion?

Guillaume

**Reply by GiudGiud** [reply]
oh in your eigenstrain model (a material property) you want to access the local position?

Well first, we do not have nodal material properties
We could develop them using the functor system if needed, though there might be a delay there
